application/app/src/main/java/uk/nhs/digital/uec/api/authentication/localstub/LocalConstants.java

```diff
@@ -10,7 +10,7 @@ private LocalConstants() {}
 
   public static final String KEY_STORE = "integTest.jks";
 
-  public static final String KEY_STORE_PASSWORD = "mypass";
```

Comment:
I wonder if we can add these local files to be ignored by the git password hook...